src/BatteryStateDisplayService.cpp

```diff
@@ -51,7 +51,8 @@ BatteryStateDisplayService::~BatteryStateDisplayService()
 void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& batteryData)
 {
     batteryStateOfChargeService_.addData(batteryData);
-
+    batteryStateOfChargeService_.isCharging();
+    qDebug()<<"Time finish charging: " <<batteryStateOfChargeService_.timeWhenChargedOrDepleted();
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
```

Comment:
I don't think QTextStream is able to print out QTIme. I got error display when I tried that